README.md

```
@@ -161,158 +156,283 @@ flowchart TD
-## 💻 Usage & Workflows
+   # Or use the executable scripts
+   # Linux/macOS: ./gws_cli --task "List my drive files"
+   # Windows: .\gws_cli.bat --task "List my drive files"
+   ```
 
 ### Examples
-Here are high-impact, realistic examples that showcase the full power of your agent — multi-service chaining, NLP complexity, and things no standard CLI can do:
+
+Here are high-impact, realistic examples that showcase the full power of your agent — multi-service chaining, NLP complexity, and things no dumb CLI can do:
 
 ***
```

markdownlint-cli2 (0.22.1)

[warning] Horizontal rule style
Expected: ---; Actual: ***

(MD035, hr-style)